tools/errorprone-checks/build.gradle.kts

```diff
@@ -18,44 +18,42 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-group = 'io.spine.tools'
+import io.spine.gradle.internal.Deps
+import io.spine.gradle.internal.Repos
+import java.net.URI
+
+group = "io.spine.tools"
 
 repositories {
-    maven { url = repos.sonatypeSnapshots }
+    maven { url = URI(Repos.sonatypeSnapshots) }
 }
 
 dependencies {
-    annotationProcessor deps.build.autoService.processor
-    compileOnly deps.build.autoService.annotations
-
-    implementation project(':base')
-    implementation project(':plugin-base')
-
-    implementation deps.build.errorProneCore
-    implementation deps.build.errorProneAnnotations
-
-    testImplementation deps.build.errorProneTestHelpers
+    annotationProcessor(Deps.build.autoService.processor)
+    compileOnly(Deps.build.autoService.annotations)
+    implementation(project(":base"))
```

Comment:
This one reads better than `"implementation"` in `build.gradle.kt` above, i.e.
```kotlin
        "implementation"(Deps.build.guava)
        "implementation"(Deps.build.checkerAnnotations)
        "implementation"(Deps.build.jsr305Annotations)
```
It is possible to use no-quote syntax there as well?